lib/LanguageServerCompletion/Model/CompletionNameImporter.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phpactor\Extension\LanguageServerCompletion\Model;
+
+use Phpactor\Extension\LanguageServerCodeTransform\Model\NameImporter\NameImporterResult;
+
+interface CompletionNameImporter
```

Comment:
I hope that is what you expected for the decoupling @dantleech  https://github.com/phpactor/language-server-phpactor-extensions/pull/35#discussion_r632349702